app/oxalis/security/WkSilhouetteEnvironment.scala

```diff
@@ -48,11 +46,20 @@ class WkSilhouetteEnvironment @Inject()(conf: WkConf,
   )
 
   val fingerprintGenerator = new DefaultFingerprintGenerator(false)
-  val idGenerator = new CompactRandomIDGenerator
-  val bearerTokenAuthenticatorDAO = new BearerTokenAuthenticatorRepository(tokenDAO)
+  val idGenerator = new CompactRandomIDGenerator()(executionContext)
```

Comment:
Why do we need to pass the executionContext explicitly now?

Reply:
I see, thanks. In this case I would prefer removing the 
`import scala.concurrent.ExecutionContext.Implicits.global`
(which I should have done already when adding the injected executionContext).
Could you do that in these instances?

Reply:
Good idea, it worked here.